tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -90,6 +89,32 @@ def single_input_to_input_cut_info(input: [str, tuple, list, PartialShape, Type,
 
     raise Exception("Unexpected object provided for input. Expected tuple, Shape, PartialShape, Type or str. Got {}".format(type(input)))
 
+def is_single_input(input: [tuple, list, str, type, Type, PartialShape, Shape]):
```

Comment:
Am I correct that this function should accept only list or tuple type because below you expect iterable object?
So we may need to correct signature of this routine:

```
def is_single_input(input: [tuple, list]):
```

Reply:
Yes, replaced with your suggestion.